python/ThirteenTeV/LFV_Z_hadronizer_RunIISummer19UL17.py

```diff
@@ -0,0 +1,48 @@
+import FWCore.ParameterSet.Config as cms
+
+externalLHEProducer = cms.EDProducer("ExternalLHEProducer",
+    args = cms.vstring('/afs/cern.ch/user/w/whajahma/public/UL19_LFV_ZtoL1L2_EWsynch/LFV_ZtoL+L-_slc7_amd64_gcc700_CMSSW_10_6_0_tarball.tar.xz'),
+    nEvents = cms.untracked.uint32(5000),
+    numberOfParameters = cms.uint32(1),
+    outputFile = cms.string('cmsgrid_final.lhe'),
+    scriptName = cms.FileInPath('GeneratorInterface/LHEInterface/data/run_generic_tarball_cvmfs.sh')
+)
+
+#Link to datacards:
```

Comment:
can you adjust this to include the proper link.

Reply:
What do you mean with proper link?